sources/update_synapse_for_appservice.sh

```diff
@@ -12,7 +12,7 @@ for f in $(ls /etc/matrix-$app/app-service/); do
 done
 
 # Set permissions
-chown matrix-$app $service_config_file
+chown --reference=$service_config_file -R /etc/matrix-$app 
```

Comment:
Well, I don't really understand why this don't work as we only edit this file. All other file should already have the good permission.

Reply:
@Josue-T  The problem is following: 
```
admin@yunohost:~$ sudo ls -l /etc/matrix-synapse/app-service
total 4
-rw------- 1 root root 411 Oct  8 08:23 mautrix_whatsapp.yaml
admin@yunohost:~$ sudo chown -R /etc/matrix-synapse/app-service --reference=/etc/matrix-synapse/conf.d/app_service.yaml
admin@yunohost:~$ sudo ls -l /etc/matrix-synapse/app-service
total 4
-rw------- 1 matrix-synapse root 411 Oct  8 08:23 mautrix_whatsapp.yaml
```